metricbeat/module/system/_meta/config.yml

```diff
@@ -1,20 +1,15 @@
 - module: system
-  period: 10s
-  metricsets:
-    - cpu
-    - load
-    - memory
-    - network
-    - process
-    - process_summary
-    #- core
-    #- diskio
-    #- socket
-  processes: ['.*']
+  # processes: ['.*']
```

Comment:
Two thins here: 

* It seemed we changed the default here from having `processes` filter enabled to disabled
* If it is commented out, the space after the `#` should be removed.

I would keep the previous config if possible.

Reply:
I commented it out because this value is already the default one, I could remove it as it is already in the reference config.

Reply:
Let's remove it.

Reply:
Ok, done.